clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -1155,14 +1162,19 @@ public ClientRequest newClientRequest(String nodeId,
         return newClientRequest(nodeId, requestBuilder, createdTimeMs, expectResponse, defaultRequestTimeoutMs, null);
     }
 
+    private int nextCorrelationId() {
+        if (SaslClientAuthenticator.isReserved(correlation)) correlation = SaslClientAuthenticator.MAX_RESERVED_CORRELATION_ID + 1;
```

Comment:
make it if-then-else since we dont't need the increment in the line below? Also split this line since we don't include the statement in the same line as `if`.

Reply:
> make it if-then-else since we dont't need the increment in the line below?

The increment make the variable has next available correlation id after evaluating the return value. 

> Also split this line since we don't include the statement in the same line as if.

Got it